client-cli/src/command/wallet_command.rs

```diff
@@ -29,6 +29,11 @@ pub enum WalletCommand {
         #[structopt(name = "name", short, long, help = "Name of wallet")]
         name: String,
     },
+    #[structopt(name = "auth_token", about = "Get authentication token")]
```

Comment:
```suggestion
    #[structopt(name = "auth-token", about = "Get authentication token")]
```
We use hyphens (`-`) everywhere in `client-cli`

Reply:
Done